### ISSUE TITLE
about tf_records_generation.py for sentinel-hub example

### ISSUE THREAD

**Issue by bobleegogogo** [developmentseed/label-maker#133]
Hello I am looking at the **Preparing Data with Sentinel Hub** example, since that label-maker also support to prepare the mask annotation of buildings as shown in this examples, do you have some suggestion how to conver the images and lables into tensorflow record for fine-tuning the pre-trained model ?

I have seen some similar python script such as  models/research/object_detection/dataset_tools/create_pet_tf_record.py, while to modify this based on label-maker results might be missing :)

**Comment by drewbo** [comment]
@bobleegogogo have you tried running the `tf_records_generation.py` script with the Sentinel Hub data? I think the major change you would need to make from the Sentinel Hub example is to us `ml_type` as `object_detection` but otherwise this should be similar to the other building detection example.

**Comment by bobleegogogo** [comment]
@drewbo Thanks for your reply. Actually, I was trying to use label-maker to prepare training samples for Mask-RCNN, this is why I need to use ml_type as segmentation. Then the tf_records_generation.py cannot meet the demand, since the training samples should contain mask and bounding boxes, not only the bounding boxes. I found some similar script here: [create_mask_rcnn_tf_record.py](https://gist.github.com/TheFlashover/34a1390a0e3e3c620e1850a72cb5826e)
